### ISSUE TITLE
Asking about range of depth map prediction value

### ISSUE THREAD

**Issue by RyanPham19092002** [donydchen/mvsplat#46]
I used my custom dataset to train your model, and I want to add depth map loss supervisor to increase the accuracy.
My depth map custom dataset has range from [0,1000] (meters). 
I want to know what range of the depth map prediction value that I can scale my custom dataset depth map to fit with those prediction and using loss function.
Thank you so much.

**Comment by donydchen** [comment]
Hi @RyanPham19092002, you might consider scaling your depth to range (1, 100) since it is the range used by the released model. However, this range is empirically set as we do not have ground truth depth in training. In your case, you can also consider setting the near and far range to better align with the ground truth depth. For more details on setting the near and far range, kindly refer to https://github.com/donydchen/mvsplat/issues/11#issuecomment-2026812577.

Besides, to train with depth loss, you might need to replace the 3DGS renderer. For more details, please refer to https://github.com/donydchen/mvsplat/issues/40#issuecomment-2199227806.

Feel free to let us know if you have any other questions. Cheers.

**Comment by RyanPham19092002** [comment]
Thank you for your support

Vào Th 6, 5 thg 7, 2024 vào lúc 17:03 Donald Chen <
***@***.***> đã viết:
